public/app/features/templating/template_srv.ts

```diff
@@ -309,22 +308,6 @@ export class TemplateSrv implements BaseTemplateSrv {
     return this.replace(target, scopedVars, 'text');
   }
 
-  fillVariableValuesForUrl = (params: any, scopedVars?: ScopedVars) => {
-    _.each(this.getVariables(), variable => {
-      if (scopedVars && scopedVars[variable.name] !== void 0) {
-        if (scopedVars[variable.name].skipUrlSync) {
-          return;
-        }
-        params['var-' + variable.name] = scopedVars[variable.name].value;
-      } else {
-        if (variable.skipUrlSync) {
-          return;
-        }
-        params['var-' + variable.name] = variableAdapters.get(variable.type).getValueForUrl(variable);
-      }
-    });
-  };
-
```

Comment:
nit: perhaps not a biggie as this is probably not used by anyone. But maybe we should deprecate this function and call the new util function `getAllVariableValuesForUrl` here instead?

Reply:
don't think we need that, this was not part of any interface we expose to plugins, and do not see any plugins using this .

Reply:
Removed